testsuite/smoke-test/src/consensus/dag/dag_fault_tolerance.rs

```diff
@@ -67,17 +67,12 @@ async fn test_no_failures() {
         5.0,
         1,
         no_failure_injection(),
-        Box::new(move |_, _, executed_rounds, executed_transactions, _, _| {
+        Box::new(move |_, _, _, executed_transactions, _, _| {
             assert!(
                 executed_transactions >= 4,
                 "no progress with active consensus, only {} transactions",
                 executed_transactions
             );
-            assert!(
-                executed_rounds >= 2,
-                "no progress with active consensus, only {} rounds",
-                executed_rounds
-            );
```

Comment:
isn't round number reset every epoch? why this assert?
@ibalajiarun @zekun000

Reply:
i thought the executed rounds are the difference between start and end round?

Reply:
with randomness enabled this is what happened in the test.
```
cycle 0 lasted 5.011 with 1 epochs, 3 rounds and 16 transactions
All at epochs: [2, 2, 2], from [1, 1, 1]
All at rounds: [6, 6, 6], from [3, 3, 3]
All at versions: [20, 20, 20], from [4, 4, 4]
cycle 0 lasted 5.010 with 1 epochs, 0 rounds and 8 transactions
All at epochs: [2, 2, 2], from [1, 1, 1]
All at rounds: [1, 1, 1], from [1, 1, 1]
All at versions: [10, 10, 10], from [2, 2, 2]
```

Reply:
without randomness:
```
cycle 0 lasted 5.011 with 1 epochs, 3 rounds and 16 transactions
All at epochs: [2, 2, 2], from [1, 1, 1]
All at rounds: [6, 6, 6], from [3, 3, 3]
All at versions: [20, 20, 20], from [4, 4, 4]
```

Reply:
i believe today it's passing the round assert just by luck?